src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -82,4 +81,16 @@
     this.deviceMetrics = deviceMetrics;
     this.userAgent = userAgent;
   }
+
+  Emulator(String deviceName, Map<String, Object> deviceMetrics, String userAgent) {
+    this.deviceName = deviceName;
+    this.deviceMetrics = deviceMetrics;
+    this.userAgent = userAgent;
+  }
+
+  Emulator(String deviceName, Map<String, Object> deviceMetrics) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "Emulator" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)